code/modules/admin/topic.dm

```diff
@@ -270,7 +270,7 @@ Status: [status ? status : "Unknown"] | Damage: [health ? health : "None"]
 			if("Cancel")
 				return
 			if("Yes")
-				location = get_turf(oldusr)
+				location = get_turf(usr.loc)
```

Comment:
Are you aware of how this will affect the behavior?

Reply:
I believed so but i didn't read above, oldusr was already the admin user, i only needed to change the line below this.